testing/testing_page.html

```diff
@@ -66,6 +66,67 @@ <h1>Widgetastic About Modal</h1>
   </div>
 
 
+<!--------------------------------- Bootstrap-Datepicker----------------------------------------->
+<!-- Special version of Bootstrap that only affects content wrapped in .bootstrap-iso -->
+<link rel="stylesheet" href="https://formden.com/static/cdn/bootstrap-iso12.css" /> 
```

Comment:
I doubt all this stuff is neccessary especially in widgitastic.patternfly. Please add two patternfly's datepicker widgets.
Furthermore, all js and css stuff for has already been added to this page above. code examples you can find in manageiq and https://www.patternfly.org/pattern-library/forms-and-controls/datepicker/#code